code/core/system/proxy-velodyne16/src/velodyne16DecoderCPC.cpp

```diff
@@ -149,9 +149,7 @@ void Velodyne16DecoderCPC::nextString(const string &payload) {
                     //Decode distance: 2 bytes. Swap the bytes, change to decimal, and divide it by 500
                     firstByte = (uint8_t)(payload.at(position));
                     secondByte = (uint8_t)(payload.at(position + 1));
-                    dataValue = ntohs(firstByte * 256 + secondByte);
-                    m_distance = dataValue / 500.0; //*2mm-->/1000 for meter
-                    m_16Sensors[sensorID]=static_cast<half>(m_distance);
+                    m_16Sensors[sensorID] = ntohs(firstByte * 256 + secondByte);
```

Comment:
What is the physical unit that your data is based on when storing it in m_16Sensors[...]? Is it cm?